aws/provider.go

```diff
@@ -570,6 +570,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_vpc_endpoint_subnet_association":          resourceAwsVpcEndpointSubnetAssociation(),
 			"aws_vpc_endpoint_service":                     resourceAwsVpcEndpointService(),
 			"aws_vpc_endpoint_service_allowed_principal":   resourceAwsVpcEndpointServiceAllowedPrincipal(),
+			"aws_vpc_secondary_ipv4_cidr_block":            resourceAwsVpcSecondaryIpv4CidrBlock(),
```

Comment:
Due to the API naming, I perosnally lean towards naming this `aws_vpc_ipv4_cidr_block_association`. I think out of context, the word `secondary` implies there can only be one of these added which is not the case (even though the EC2 User Guide refers to them that way. Either way it can be confusing. 😕

Reply:
Will do.